src/Monitoring/Monitoring.ArcadeServices/dashboard/arcade-services/arcadeAvailability.dashboard.json

```diff
@@ -2866,6 +2866,48 @@
       }
     },
     {
+      "alert": {
+        "alertRuleTags": {
+          "NotificationId": "b87805b575d141e4b2b6d258f5e3a79e"
+        },
+        "conditions": [
+          {
+            "evaluator": {
+              "params": [
+                10
```

Comment:
How was the alert level of "average greater than 10" decided?

Reply:
That's the red line that was picked on the graph. It's easy to change if it should be something else.